inc/customizer.php

```diff
@@ -1042,12 +1043,18 @@ function estore_customize_partial_blogdescription() {
 	bloginfo( 'description' );
 }
 
-// Function for top eader text selective refresh support
+// Function for top header text selective refresh support
 function estore_bar_text() {
```

Comment:
Since you're trying to make this a deprecated function and change its name, you need to change it to the new name.